src/sentry/api/endpoints/project_codeowners.py

```diff
@@ -151,12 +159,14 @@ def update(self, instance, validated_data):
 
 
 class ProjectCodeOwnersMixin:
-    def has_feature(self, request, project):
-        return features.has(
-            "organizations:import-codeowners", project.organization, actor=request.user
+    def has_feature(self, request: Any, project: Project) -> bool:
```

Comment:
that's a django request afaik

Reply:
Thanks, I was having trouble figuring out which `Request` class to use.